packages/react/src/infrastructure/theme/types.ts

```diff
@@ -0,0 +1,151 @@
+export enum LogoColor {
+  BLACK,
+  WHITE,
+}
+
+export enum ConnectWalletDefaultIconType {
+  WITH_COLOR,
+  WHITE,
+}
+
+export enum ThemeType {
+  DARK_MODE = 'DarkMode',
+  LIGHT_MODE = 'LightMode',
+}
+
+export type ThemeContextType = {
+  darkModeFsEnabled: boolean;
+  theme: ThemeType;
+  toggleTheme: () => void;
+};
+export interface Theme {
```

Comment:
is the theming applied because I did not see it in many CSS components

Reply:
Theme will be use in global css changes in project.